packages/app/tsconfig.node.json

```diff
@@ -1,9 +1,9 @@
 {
   "compilerOptions": {
     "tsBuildInfoFile": "./node_modules/.tmp/tsconfig.node.tsbuildinfo",
-    "target": "ES2022",
+    "target": "ES2023",
     "module": "nodenext",
-    "lib": ["ES2022"],
+    "lib": ["ES2023"],
```

Comment:
Slightly unrelated, but the node version we use supports ES2023, so I thought this was more correct.